### ISSUE TITLE
[Question] Can we have html tags in translation strings?

### ISSUE THREAD

**Issue by sanket-thotange** [biesbjerg/ngx-translate-extract#262]
`{{ 'Alert Count <strong>{count}</strong>' | translate: {count: 4} }}`

**Comment by alebx** [comment]
Yes, but you need to render the content as html e.g. `<div [innerHTML]="'Alert Count <strong>{count}</strong>' | translate: {count: 4}"></div>`. For some HTML tags you might also need to use/bypass the [DomSanitizer](https://angular.io/api/platform-browser/DomSanitizer).
